### PR TITLE
E2e/feature/ri 5072 docker encryption

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/helpers/database-scripts.ts [hunk 1]
```diff
@@ -3,70 +3,111 @@ import { workingDirectory } from '../helpers/conf';
...
 }
+/**
+ * Add new database parameters
+ * @param tableName The name of table in DB
+     * @param columnName The name of column in table
+     * @param rowValue Value to update in table
+     * @param conditionColumnName The name of the column to search
+     * @param conditionColumnValue The value to match in the column
+ */
+export type DbTableParameters = {
```
Comment (mariasergeenko) [inline tests/e2e/helpers/database-scripts.ts:107]: what is the different between conditionColumnName and columnName?
and pleases clarify me, how exactly we should specify what row should be undated?

File: tests/e2e/tests/web/critical-path/a-first-start-form/user-agreements-form.e2e.ts [hunk 2]
```diff
@@ -22,8 +22,7 @@ test('Verify that user should accept User Agreements to continue working with th
     await t.expect(userAgreementDialog.submitButton.hasAttribute('disabled')).ok('Submit button not disabled by default');
     await t.expect(myRedisDatabasePage.AddRedisDatabase.addDatabaseManually.exists).notOk('User can\'t add a database');
 });
-// Skipped due to docker currently doesn't support encryption https://redislabs.atlassian.net/browse/RI-5036
-test.skip('Verify that the encryption enabled by default and specific message', async t => {
+test.only('Verify that the encryption enabled by default and specific message', async t => {
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/a-first-start-form/user-agreements-form.e2e.ts:25]: only))))

File: tests/e2e/tests/web/critical-path/database/encryption.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await databaseHelper.acceptLicenseTerms();
+        await databaseAPIRequests.addNewStandaloneDatabaseApi(ossStandaloneTlsConfig);
+        await myRedisDatabasePage.reloadPage();
+    })
+    .afterEach(async() => {
+        await databaseHelper.deleteDatabase(ossStandaloneTlsConfig.databaseName);
+    });
+test.only('Verify that data encrypted using KEY', async t => {
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/database/encryption.e2e.ts:34]: only

File: tests/e2e/helpers/database-scripts.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/helpers/database-scripts.ts:80]: renamed to conditionWhereColumnName and conditionWhereColumnValue

File: tests/e2e/tests/web/critical-path/a-first-start-form/user-agreements-form.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/a-first-start-form/user-agreements-form.e2e.ts]: removed

File: tests/e2e/tests/web/critical-path/database/encryption.e2e.ts (hunk 3 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/database/encryption.e2e.ts]: removed

**Review by mariasergeenko — APPROVED** [review]

**Pull request merged**